ts/src/base/Exchange.ts

```diff
@@ -2520,7 +2521,11 @@ export default class Exchange {
         } catch (e) {
             errorMessage = e.toString ();
         }
-        throw new NotSupported (this.id + ' ' + method + '() failed to fetch correct data from website. Probably webpage markup has been changed, breaking the page custom parser.' + errorMessage);
+        if (muteOnFailure) {
+            return undefined;
+        } else {
+            throw new NotSupported (this.id + ' ' + method + '() failed to fetch correct data from website. Probably webpage markup has been changed, breaking the page custom parser.' + errorMessage);
```

Comment:
hm, whe I remove the errormessage assignment from `catch` then we are left with empty block, causing transpiled code error in python, like:
```
    except Exception as e:
        # while here should be something, but it's empty line here because of our incomplete transpiler, causing python bug
```

Reply:
if it's not priority, we might merge without that change and I will try to remove that errormessage later

Reply:
@ttodua I think we can keep the throw just with a different exception message (something very generic), and maybe using `BadResponse` instead

Reply:
promise - this is last question about this PR :D 
is the last commit ok (to leave the possibility for option to set that as option, so in rare cases we could get some debug info from user (i think in some extreme cases it might help to remotely debug whatever important issue).

Reply:
@ttodua sorry i think this is too much 😅, 3 different options just for controlling things on fetchCurrencies we're overcomplicating it a bit IMO

Reply:
:))
oky (but this is not just for fetchCurrencies, some exchanges do use it for fetchMarkets too and also in other cases, it might not have hurted :)) but sure sure let me push changes

Reply:
if something, we can ask user to enable verbose - so that would be enough.

Reply:
done. lmk if pr looks good now. i've tested kucoin in case of intended failure, and it still works (just networks are empty as expected inside currencies)